isis/appdata/import/cassis.lbl.tpl

```diff
@@ -1,3 +1,8 @@
+{% set PO=Product_Observational %}
```

Comment:
This is a great idea! Makes it much easier to read.

Reply:
We may need to be more specific for this file name. There are likely more cassis templates to come.